### PR TITLE
[Merged by Bors] - chore(NormedSpace/Basic): move some theorems to `NormedSpace.Real`

### PR DESCRIPTION
**Pull request by urkud**
This way we don't switch between general normed spaces and real normed spaces
back and forth throughout the file.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Analysis/NormedSpace/Basic.lean [hunk 1]
```diff
@@ -113,63 +91,6 @@ theorem Filter.IsBoundedUnder.smul_tendsto_zero {f : ι → α} {g : ι → E} {
     (norm_smul_le y x).trans_eq (mul_comm _ _)
 #align filter.is_bounded_under.smul_tendsto_zero Filter.IsBoundedUnder.smul_tendsto_zero
 
-theorem closure_ball [NormedSpace ℝ E] (x : E) {r : ℝ} (hr : r ≠ 0) :
```
Comment (sgouezel) [inline Mathlib/Analysis/NormedSpace/Basic.lean:116]: This result and the next ones are not specifically about real vector spaces, they would work on any DenselyNormedField.

Comment (urkud) [reply Mathlib/Analysis/NormedSpace/Basic.lean:116]: How do I prove that there exists `c : K` such that `‖c‖ < 1` and `‖1 - c‖ < ε`?

Comment (sgouezel) [reply Mathlib/Analysis/NormedSpace/Basic.lean:116]: Argh, you're right, it's not true in general! Then I'm happy with your PR. Sorry for the noise!

**Comment by sgouezel** [comment]
bors r+
Thanks!

**Pull request closed without merging**